### PR TITLE
feat(stripe): create attributes stripe

### PR DESCRIPTION
**Pull request by ptmccrary**

### CODE REVIEW

File: templates/stripes/stripe-attributes.php [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<div class="<?= $this->e($type, 'wrapperClasses') ?>">
+  <div class="vssl-stripe-column">
+    <div class="vssl-stripe--attributes-fieldwrap">
+      <?php foreach ($values as $key => $value) : ?>
+        <div class="vssl-stripe--attributes-field vssl-stripe--attributes--<?= $key ?>">
```
Comment (joshuabryant) [inline templates/stripes/stripe-attributes.php:5]: Second class should probably be `vssl-stripe--attributes-field--<?= $key ?>`, yeh?

File: templates/stripes/stripe-attributes.php [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<div class="<?= $this->e($type, 'wrapperClasses') ?>">
+  <div class="vssl-stripe-column">
+    <div class="vssl-stripe--attributes-fieldwrap">
```
Comment (joshuabryant) [inline templates/stripes/stripe-attributes.php:3]: Wrap an `if (!empty($values))` around this?

File: templates/stripes/stripe-attributes.php [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+      <?php foreach ($values as $key => $value) : ?>
+        <div class="vssl-stripe--attributes-field vssl-stripe--attributes--<?= $key ?>">
+          <div class="vssl-stripe--attributes-label"><?= $key ?>:</div>
+          <div class="vssl-stripe--attributes-value">
+            <?= $value['html'] ?>
```
Comment (joshuabryant) [inline templates/stripes/stripe-attributes.php:8]: Change to `!empty($value['html']) ? $value['html'] : ''`, maybe?

**Review by joshuabryant — CHANGES_REQUESTED** [review]
Overall, add a few lines of defensive code, and we'll be ready to ship.

File: templates/stripes/stripe-attributes.php (hunk 1 above)
Comment (joshuabryant) [reply templates/stripes/stripe-attributes.php:5]: Ugh, yeh. We've ended up with a strange variation on BEM-style classes that eschews underscores. Based on other templates, I'd recommend many more double dashes in this code.

For example:
```
<div class="vssl-stripe--attributes--fieldwrap">
  <div class="vssl-stripe--attributes--field vssl-stripe--attributes--<?= $key ?>-field">
    ...
```

File: templates/stripes/stripe-attributes.php [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+            <div class="vssl-stripe--attributes--value">
+              <?= $value['html'] ?>
+            </div>
+          </div>
+          <?php endif; ?>
+        <?php endforeach; ?>
+      </div>
+    </div>
+  </div>
+<?php endif; ?>
```
Comment (merricks) [inline templates/stripes/stripe-attributes.php:18]: Add a newline to the end of this file :)

File: templates/stripes/stripe-attributes.php [hunk 5]
```diff
@@ -0,0 +1,18 @@
+<?php if (!empty($values)) : ?>
```
Comment (merricks) [inline templates/stripes/stripe-attributes.php:1]: Will this still pass if all the fields are empty but the labels/keys aren't?

File: templates/stripes/stripe-attributes.php [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+  <div class="<?= $this->e($type, 'wrapperClasses') ?>">
+    <div class="vssl-stripe-column">
+      <div class="vssl-stripe--attributes--fieldwrap">
+        <?php foreach ($values as $key => $value) : ?>
+          <?php if (!empty($value['html']) ? $value['html'] : '') : ?>
```
Comment (merricks) [inline templates/stripes/stripe-attributes.php:6]: `<?php if (!empty($value['html'])) : ?>`

File: templates/stripes/stripe-attributes.php [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+          <div class="vssl-stripe--attributes--field vssl-stripe--attributes--field--<?= $key ?>">
+            <div class="vssl-stripe--attributes--label"><?= $key ?>:</div>
+            <div class="vssl-stripe--attributes--value">
+              <?= $value['html'] ?>
```
Comment (merricks) [inline templates/stripes/stripe-attributes.php:10]: Strip tags here (constrain to inline, or inline + `<address>`)—we don't want multiple paragraphs, for example.

Comment (merricks) [reply templates/stripes/stripe-attributes.php:10]: (And could combine with lines 9 & 11 if it doesn't get too much longer :) )

**Review by merricks — APPROVED** [review]

**Pull request merged**